src/Microsoft.OData.Core/JsonLight/ODataJsonLightOutputContext.cs

```diff
@@ -97,7 +102,11 @@ internal sealed class ODataJsonLightOutputContext : ODataOutputContext
                 }
                 else
                 {
+#if NETSTANDARD1_1
                     this.asynchronousOutputStream = new AsyncBufferedStream(this.messageOutputStream);
+#else 
+                    this.asynchronousOutputStream = new BufferedStream(this.messageOutputStream, 84000);

```

Comment:
Since this buffer size is repeated in many places, I think it would be better to store it in a constant.